tests/foreman/ui_airgun/test_discoveredhost.py

```diff
@@ -174,6 +174,7 @@ def test_positive_pxe_less_with_dhcp_unattended(session, provisioning_env):
             assert discovered_host_values['Name'] == host_name
 
 
+@skip_if_bug_open('bugzilla', 1666217)
```

Comment:
if there test cases worked before and now we skip them because of 1666217 then that BZ is a regression?

Reply:
Yeah, the bug should be regression. Added that to bug.

Reply:
Anyways this bug has been closed duplicate of bug - https://bugzilla.redhat.com/show_bug.cgi?id=1665471

Reply:
Then please change a PR :)

Reply:
@oshtaier , I changed.

Reply:
nvm, sometimes github does not refresh pages correctly